### PR TITLE
added support for ubuntu release 17.04

### PR DESCRIPTION
**Pull request by pfichtner**

### CODE REVIEW

File: tpl/vagrantfile-ubuntu1704-desktop.tpl [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        v.gui = true
+        v.customize ["modifyvm", :id, "--memory", 1024]
+        v.customize ["modifyvm", :id, "--cpus", 1]
+        v.customize ["modifyvm", :id, "--vram", "256"]
+        v.customize ["setextradata", "global", "GUI/MaxGuestResolution", "any"]
+        v.customize ["setextradata", :id, "CustomVideoMode1", "1024x768x32"]
+        v.customize ["modifyvm", :id, "--ioapic", "on"]
+        v.customize ["modifyvm", :id, "--rtcuseutc", "on"]
+        v.customize ["modifyvm", :id, "--accelerate3d", "on"]
+        v.customize ["modifyvm", :id, "--clipboard", "bidirectional"]
```
Comment (inetbiz) [inline tpl/vagrantfile-ubuntu1704-desktop.tpl]: @pfichtner might I suggest also adding:
```
   v.customize ["modifyvm", :id, "--audio", "none"]
   v.customize ["modifyvm", :id, "--draganddrop", "hosttoguest"]
```

File: ubuntu1704-desktop.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{
+  "_comment": "Build with `packer build -var-file=ubuntu1704-desktop.json ubuntu.json`",
+  "vm_name": "ubuntu1704-desktop",
+  "desktop": "true",
+  "locale": "en_US.UTF-8",
+  "cpus": "1",
+  "disk_size": "130048",
```
Comment (inetbiz) [inline ubuntu1704-desktop.json]: This attribute confused me. I'm not sure if it's the size of the iso or the size you want the VM?

File: tpl/vagrantfile-ubuntu1704-desktop.tpl (hunk 1 above)
Comment (pfichtner) [inline tpl/vagrantfile-ubuntu1704-desktop.tpl]: none of the six other ubuntu desktop templates have the settings set. I did just copy from 1610-desktop and did change values from 1610 to 1704. So this suggestion should be a general patch where these attributes will be added to all seven templates.

File: ubuntu1704-desktop.json (hunk 2 above)
Comment (pfichtner) [inline ubuntu1704-desktop.json]: Values are the same like in the other ubuntu json files. So if they are shady here they are shady in all the files too. Issue should be discussed with the people responsible for the values and should be fixed in all the other files too.

Comment (inetbiz) [inline ubuntu1704-desktop.json]: Not shady. I'm new to vagrant. And, just now learning about this project. So, is there a forum or other place to discuss and learn if that value is the size of vm or size of iso?

Comment (pfichtner) [inline ubuntu1704-desktop.json]: It's the size of the hard_disc (VM). 
The discussion you did start is a discussion regarding an open pull request. So please do not comment the pull request as long as the topic isn't the pull request.

**Pull request merged**